### PR TITLE
feat: add pages

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Pages**
- Home Page
- Single video page ( Watch page )
- Search Page
- Login Page
- Signup Page

> added routes for different pages

### CODE REVIEW

File: src/components/Sidebar/Sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    display: flex;
+    flex-direction: column;
+
+    width: 90px;
+    height: 90vh;
+    padding-top: 2rem;
+    border-right: 1px solid var(--white-color);
+    transition: transform 0.3s ease-in;
+
+    li {
```
Comment (sharathnair9999) [inline src/components/Sidebar/Sidebar.scss]: better to specify the classes and style them

**Review by sharathnair9999 — APPROVED** [review]
overall code looks clean

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <FaBars
+                    className="revisit__header__menu"
+                    size={26}
+                    onClick={() => setShowNav((prevValue) => !prevValue)}
+                />
+                <p className="brand__name">Revisit</p>
+            </div>
+
+            <div className="brand">
+                {/* <img
```
Comment (ashishnnnnn) [inline src/components/Navbar/Navbar.jsx:20]: Commented Code Can Be Omitted ..If not required..

File: src/pages/Search/Search.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import VideoCard from "../../components/VideoCard/VideoCard";
+
+const Search = () => {
+    return (
+        <div>
```
Comment (ashishnnnnn) [inline src/pages/Search/Search.jsx:6]: Liked This Search Implementation

**Review by ashishnnnnn — APPROVED** [review]
Nice Usage of SCSS ..
Code seems pretty much good to merge :+1:

**Pull request merged**